python/test/test_datasetslice.py

```diff
@@ -177,6 +174,9 @@ def test_correct_temporaries(self):
         d1 = d1[Dim.X, 1:2]
         self.assertEqual(list(d1[Data.Value, "A"].data), [5.0, 6.0, 7.0, 8.0])
 
+    def test_set_dataset_slice_items(self):
+        self._d[Data.Value, "a"][Dim.X, 0:2] += self._d[Data.Value, "b"][Dim.X, 1:3]
+        self.assertEqual(list(self._d[Data.Value, "a"].data), [1, 3, 2, 3, 4, 5, 6, 7, 8, 9])
```

Comment:
Maybe it is worth do the copy of `_d` to play with it, I mean _d can be used in other tests.